scripts/vote_2025_05_11.py

```diff
@@ -0,0 +1,200 @@
+HASH_CONSENSUS_FOR_CS_FEE_ORACLE_QUORUM = 5
+
+# Oracles members
+old_oracle_member_to_remove = "0x140bd8fbdc884f48da7cb1c09be8a2fadfea776e"
+new_oracle_member_to_add = "0xe050818F12D40b4ac8bf99a9f9F9517b07428D58"
+
+
+description = """
+**Emergency rotation of compromised Chorus One oracle.**
+
+Rotate Chorus One's address in the Lido on Ethereum Oracle set. Full context on [Research forum]().
+"""
+
+def encode_remove_accounting_oracle_member(member: str, quorum: int) -> Tuple[str, str]:
+    hash_consensus = contracts.hash_consensus_for_accounting_oracle
+
+    return (hash_consensus.address, hash_consensus.removeMember.encode_input(member, quorum))
+
+
+def encode_remove_validators_exit_bus_oracle_member(member: str, quorum: int) -> Tuple[str, str]:
+    hash_consensus = contracts.hash_consensus_for_validators_exit_bus_oracle
+
+    return (hash_consensus.address, hash_consensus.removeMember.encode_input(member, quorum))
+
+
+def encode_remove_validators_cs_fee_oracle_member(member: str, quorum: int) -> Tuple[str, str]:
+    hash_consensus = contracts.csm_hash_consensus
+
+    return (hash_consensus.address, hash_consensus.removeMember.encode_input(member, quorum))
+
+
+def encode_add_accounting_oracle_member(member: str, quorum: int) -> Tuple[str, str]:
+    hash_consensus = contracts.hash_consensus_for_accounting_oracle
+
+    return (hash_consensus.address, hash_consensus.addMember.encode_input(member, quorum))
+
+
+def encode_add_validators_exit_bus_oracle_member(member: str, quorum: int) -> Tuple[str, str]:
+    hash_consensus = contracts.hash_consensus_for_validators_exit_bus_oracle
+
+    return (hash_consensus.address, hash_consensus.addMember.encode_input(member, quorum))
+
+
+def encode_add_cs_fee_oracle_member(member: str, quorum: int) -> Tuple[str, str]:
+    hash_consensus = contracts.csm_hash_consensus
+
+    return (hash_consensus.address, hash_consensus.addMember.encode_input(member, quorum))
+
+
+def start_vote(tx_params: Dict[str, str], silent: bool) -> Tuple[int, Optional[TransactionReceipt]]:
+    """Prepare and run voting."""
+
+    vote_desc_items, call_script_items = zip(
+        # Chorus One oracle rotation
+        (
+            "1. Remove the oracle member with address 0x140bd8fbdc884f48da7cb1c09be8a2fadfea776e from HashConsensus for AccountingOracle",
+            agent_forward(
+                [
+                    encode_remove_accounting_oracle_member(
+                        old_oracle_member_to_remove, HASH_CONSENSUS_FOR_ACCOUNTING_ORACLE_QUORUM
+                    )
+                ],
+            ),
+        ),
+        (
+            "2. Remove the oracle member with address 0x140bd8fbdc884f48da7cb1c09be8a2fadfea776e from HashConsensus for ValidatorsExitBusOracle",
+            agent_forward(
+                [
+                    encode_remove_validators_exit_bus_oracle_member(
+                        old_oracle_member_to_remove, HASH_CONSENSUS_FOR_ACCOUNTING_ORACLE_QUORUM
```

Comment:
```suggestion
                        old_oracle_member_to_remove, HASH_CONSENSUS_FOR_VALIDATORS_EXIT_BUS_ORACLE_QUORUM
```

Reply:
Consider having only one constant for all oracles, since the oracle set is synchronized